Search.java

```diff
@@ -0,0 +1,59 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+class Search {
+
+    public static void main(String[] args) {
+        boolean case_sensitivity = true;
+        Scanner file_scanner;
+        String input_file = args[1];
+        String input_string = args[0];
```

Comment:
"input_string" might be too generic name; "input_pattern" would be more clear in my opinion!